SDL2pp/Window.cc

```diff
@@ -264,6 +264,32 @@ Window& Window::SetResizable(bool resizable) {
 	SDL_SetWindowResizable(window_, resizable ? SDL_TRUE : SDL_FALSE);
 	return *this;
 }
+
+Window& Window::SetModal(const Window& parent) {
+	if (SDL_SetWindowModalFor(window_, parent.window_) != 0) {
+		throw SDL2pp::Exception("SDL_SetWindowModalFor");
+	}
+	
+	return *this;
+}
+
+std::map<Window::Border, int> Window::GetBordersSize(std::initializer_list<Window::Border> borders) const {
```

Comment:
Umm, no. No, no, definitely not a map, and not an initializer list. Just return a tuple/struct (probably struct is better since it's easy to mix up the sides) of 4 ints, user is free to use only values he needs.